### PR TITLE
[TEST] Export voucher codes

### PR DESCRIPTION
**Pull request by yellowee**
I want to merge this change because it covers [CORE_0925](https://saleor.testmo.net/repositories/5?group_id=131&sort_column=repository_cases:custom_tc_id&sort_direction=desc&case_id=4753)

# Impact

- [ ] New migrations
- [ ] New/Updated API fields or mutations
- [ ] Deprecated API fields or mutations
- [ ] Removed API types, fields, or mutations

# Docs

- [ ] Link to documentation:

# Pull Request Checklist

- [ ] Privileged queries and mutations are either absent or guarded by proper permission checks
- [ ] Database queries are optimized and the number of queries is constant
- [ ] Database migrations are either absent or optimized for zero downtime
- [ ] The changes are covered by test cases

### CODE REVIEW

**Comment by IKarbowiak** [comment]
You should use temporary directory to not create actual files. Try using `media_root` fixture

File: saleor/tests/e2e/vouchers/test_export_voucher_codes.py [hunk 1]
```diff
@@ -0,0 +1,495 @@
...
+        _channel_slug,
+        _shipping_method_id,
+    ) = prepare_shop(e2e_staff_api_client)
+
+    voucher_ids, _voucher_code_ids = create_vouchers_with_multiple_codes(
+        e2e_staff_api_client, channel_id
+    )
+
+    voucher_id_array_index = [voucher_ids[voucher_id_array_index]]
+
```
Comment (szczecha) [inline saleor/tests/e2e/vouchers/test_export_voucher_codes.py:106]: `# Step 1 - Export voucher codes and check status`

**Review by szczecha — APPROVED** [review]

File: saleor/tests/e2e/vouchers/test_export_voucher_codes.py [hunk 2]
```diff
@@ -0,0 +1,496 @@
...
+    e2e_staff_api_client,
+    permission_manage_channels,
+    permission_manage_shipping,
+    permission_manage_products,
+    permission_manage_discounts,
+    permission_manage_checkouts,
+    file_type,
+    voucher_id_array_index,
+    expected_status,
+):
```
Comment (fowczarek) [inline saleor/tests/e2e/vouchers/test_export_voucher_codes.py:83]: Maybe we could use exisitng `media_root` fixture? 
```suggestion
@pytest.mark.e2e
@pytest.mark.parametrize(
    ("file_type", "voucher_id_array_index", "expected_status"),
    [
        ("CSV", 0, "SUCCESS"),
        ("XLSX", 2, "SUCCESS"),
    ],
)
def test_export_valid_voucher_ids_CORE_0925(
    e2e_staff_api_client,
    permission_manage_channels,
    permission_manage_shipping,
    permission_manage_products,
    permission_manage_discounts,
    permission_manage_checkouts,
    file_type,
    voucher_id_array_index,
    expected_status,
    media_root
):
```

File: saleor/tests/e2e/vouchers/test_export_voucher_codes.py [hunk 3]
```diff
@@ -0,0 +1,496 @@
...
+        "fileType": file_type,
+        "voucherId": voucher_id_array_index,
+    }
+    response = export_voucher_codes(e2e_staff_api_client, input_data)
+
+    assert response["status"] == expected_status
+
+
+@pytest.mark.e2e
+@pytest.mark.parametrize(
```
Comment (fowczarek) [inline saleor/tests/e2e/vouchers/test_export_voucher_codes.py:118]: Both tests have the same error. There is no reason to store errors in parametrize.

File: saleor/tests/e2e/vouchers/test_export_voucher_codes.py [hunk 4]
```diff
@@ -0,0 +1,496 @@
...
+        "fileType": file_type,
+        "ids": [invalid_voucher_code],
+    }
+    response = raw_export_voucher_codes(e2e_staff_api_client, input_data)
+
+    assert response["data"]["exportVoucherCodes"]["errors"] == expected_errors
+
+
+@pytest.mark.e2e
+@pytest.mark.parametrize(
```
Comment (fowczarek) [inline saleor/tests/e2e/vouchers/test_export_voucher_codes.py:375]: Both tests have the same error. There is no reason to store errors in parametrize.

**Review by korycins — APPROVED** [review]

**Review by fowczarek — APPROVED** [review]

**Pull request merged**